pylops/avo/avo.py

```diff
@@ -627,7 +627,7 @@ def _matvec(self, x):
                 + self.spatdims
             )
         y = self.ncp.sum(self.G * x[:, :, self.ncp.newaxis], axis=1)
-        return y
+        return y.ravel()
```

Comment:
We should consider making this operator consistent with others, using `dims` and `dimsd` :)

Also, why is this needed now. In the current dev:

```
(pylops.avo.AVOLinearModelling(np.ones(5),nt0=100) * np.ones(300)).shape
Out[21]: (500,)
```
and
```
(pylops.avo.AVOLinearModelling(np.ones(5),nt0=100, spatdims=20) * np.ones(300*20)).shape
Out[23]: (10000,)
```